Model/Media.php

```diff
@@ -630,7 +629,7 @@ public function getCategory()
     /**
      * @param CategoryInterface $category|null
      */
-    public function setCategory(CategoryInterface $category = null)
+    public function setCategory($category = null)
```

Comment:
It's NOT a BC Break! The BC break was caused in https://github.com/sonata-project/SonataMediaBundle/commit/0dbc19f0 . This is removing the couple of SonataMedia to SonataClassification... This accept a category and not *only* the CategoryInterface from categoryclassification...